### DISCUSSION TITLE
Is this normal on Git for Windows?

### DISCUSSION THREAD

**Discussion by bowstones** [community/community#22803] [New to GitHub]
<p>Just trying to commit to my GH repo from the command line on Windows, it’s the first time I’ve done this.</p>
<p>After setting up my commit I did <code>git push --set-upstream origin master</code> and this window appeared:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/ec89069714f78ef160df4b6c415a195db4468af7.png" data-download-href="/uploads/short-url/xKug4QEsvhoZ660Zw1yIVTVql6f.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/34344190/181073174-497215ea-0ebc-46aa-8e8d-770bfbb9518b.png" alt="image" data-base62-sha1="xKug4QEsvhoZ660Zw1yIVTVql6f" width="500" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/ec89069714f78ef160df4b6c415a195db4468af7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">510×509 12.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Is that meant to happen, and am I OK to put my GitHub login details into it? Or is there something weird or concerning going on here?</p>

**Comment by Gisgar3** [comment, marked as answer]
<p>Hey, welcome to the <strong>GitHub Support Community</strong>! This <strong>is</strong> normal behavior, as the system doesn’t have any saved Git account credentials. Once you fill in your information, you should not need to fill it in again afterwards.</p>
<p>Hope this helps!</p>

**Comment by bowstones** [comment]
<p>Thanks - I’ll go ahead and log in there then :+1:</p>
